src/static/bitbucket/atlassian-connect.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "Atlaskid",
+  "description": "Addon to display a 'release queue' panel in PRs",
+  "baseUrl": "https://lbatch.ngrok.io/",
+  "key": "atlaskid-addon",
+  "vendor": {
+    "name": "Luke Batchelor"
+  },
+  "scopes": ["pullrequest"],
+  "authentication": {
+    "type": "jwt"
+  },
+  "contexts": ["personal"],
+  "modules": {
+    "webPanels": [
+      {
+        "weight": 100,
+        "tooltip": {
+          "value": "Packages to be released in this PR"
+        },
+        "key": "atlaskid-addon-panel",
+        "name": {
+          "value": "Release queue"
+        },
+        "url":
+          "/bitbucket/index.html?repoOwner={pullrequest.source.repository.owner.username}&repoSlug={pullrequest.source.repository.name}&pullRequestId={pullrequest.id}&username={user.username}&userUuid={user.uuid}&state={pullrequest.state}&commit={pullrequest.source.commit.hash}&title={pullrequest.title}",
+        "location": "org.bitbucket.pullrequest.overview.informationPanel",
+        "conditions": [
+          {
+            "condition": "has_permission",
+            "params": {
+              "permission": "write"
+            }
+          }
+        ]
+      }
+    ],
+    "webItems": [
+      {
+        "url": "/ready-to-land",
+        "name": {
+          "value": "Ready to land!"
+        },
+        "location": "org.bitbucket.pullrequest.summary.actions",
+        "key": "atlaskid-addon-land-button",
+        "params": {
+          "auiIcon": "aui-iconfont-homepage"
+        }
+      }
+    ],
+    "webhooks": [
+      {
```

Comment:
Can remove this extra webhook too. Unless we want to more info when tracking the land builds as they get created.